### PR TITLE
Add missing configuration options to grpc readme

### PR DESCRIPTION
**Pull request by laurit**

### CODE REVIEW

File: instrumentation/grpc-1.6/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Settings for the gRPC instrumentation
 
-| System property                                          | Type    | Default | Description                                         |
-| -------------------------------------------------------- | ------- | ------- | --------------------------------------------------- |
-| `otel.instrumentation.grpc.experimental-span-attributes` | Boolean | `false` | Enable the capture of experimental span attributes. |
+| System property                                             | Type    | Default | Description                                                                               |
+|-------------------------------------------------------------|---------|---------|-------------------------------------------------------------------------------------------|
+| `otel.instrumentation.grpc.experimental-span-attributes`    | Boolean | `false` | Enable the capture of experimental span attributes.                                       |
```
Comment (jeanbisutti) [inline instrumentation/grpc-1.6/README.md:5]:
```suggestion
| `otel.instrumentation.grpc.experimental-span-attributes`    | Boolean | `false` | Enable the capture all of experimental span attributes.                                       |
``` ?
The reading of the two following properties may be easier.

File: instrumentation/grpc-1.6/README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+| `otel.instrumentation.grpc.capture-metadata.client.request` | String  |         | Comma separated list of request metadata keys that should be captured as span attributes. |
```
Comment (jeanbisutti) [inline instrumentation/grpc-1.6/README.md:6]: Something like this to insist that it is for client?
```suggestion
| `otel.instrumentation.grpc.capture-metadata.client.request` | String  |         | Comma separated list of request metadata keys on client side that should be captured as span attributes. |
```

File: instrumentation/grpc-1.6/README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
+| `otel.instrumentation.grpc.capture-metadata.server.request` | String  |         | Comma separated list of request metadata keys that should be captured as span attributes. |
```
Comment (jeanbisutti) [inline instrumentation/grpc-1.6/README.md:7]: Something like this to insist that it is for server?
```suggestion
| `otel.instrumentation.grpc.capture-metadata.server.request` | String  |         | Comma separated list of request metadata keys on server side that should be captured as span attributes. |
```

File: instrumentation/grpc-1.6/README.md (hunk 1 above)
Comment (laurit) [reply instrumentation/grpc-1.6/README.md:5]: We use the same wording almost everywhere where we have a flag that ends with `experimental-span-attributes`. If you wish to change it it is best to change it everywhere and do it in a separate pr.

**Review by jeanbisutti — APPROVED** [review]

**Pull request merged**